src/kshell.c

```diff
@@ -22,14 +23,15 @@ static int print_directory( char *d, int length )
 	return 0;
 }
 
-static int mount_cd( int unit )
+static int mount_cd( int unit , char *fs_type )
 {
-	struct fs *cdrom = fs_get("cdrom");
-	struct fs_volume *v = fs_volume_mount(cdrom, unit);
+	struct fs *fs = fs_get(fs_type);
```

Comment:
Must check for failure of fs_get, it's very easy for the user to type a filesystem that doesn't exist, and crash the system.